imagepipeline-base/src/main/java/com/facebook/imageformat/ImageFormatChecker.java

```diff
@@ -50,19 +50,20 @@ public ImageFormat determineImageFormat(final InputStream is) throws IOException
     final byte[] imageHeaderBytes = new byte[mMaxHeaderLength];
     final int headerSize = readHeaderFromStream(mMaxHeaderLength, is, imageHeaderBytes);
 
+    ImageFormat format = mDefaultFormatChecker.determineFormat(imageHeaderBytes, headerSize);
```

Comment:
Unfortunately, our pull request import will squash all of your commits into one, so everything will be in the same changeset. Could you open a separate pull request for changing the order so that the commit title and blame make more sense? Thanks!